src/libraries/System.Text.Json/ref/System.Text.Json.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
+  <PropertyGroup>  
```

Comment:
nit: Why do we have this trailing space in this change?

Reply:
Leftover from merging conflict